src/js/components/validateData/validateValues/ValidateValuesErrorReport.jsx

```diff
@@ -66,7 +66,7 @@ export default class ValidateValuesErrorReport extends React.Component {
 
             let description = item.error_description;
             if (item.error_name == 'rule_failed') {
-                description = item.rule_failed;
+                description = ['Rule ' + item.original_label + ': ' + item.rule_failed];
```

Comment:
Right, but the actual text that is returned in `item.rule_failed` is different. Because we didn't have labels in this one originally it said `Failed rule: RULE TEXT` so now, with the change, it says `Rule LABEL: Failed rule: RULE TEXT`.

It seems redundant especially since in cross-file validations it just says `Rule LABEL: RULE TEXT`